content/en/docs/concepts/services-networking/service.md

```diff
@@ -42,14 +43,13 @@ to, so that the frontend can use the backend part of the workload?
 
 Enter _Services_.
 
-## Service resources {#service-resource}
```

Comment:
Could this removal introducing some dangling links?

Reply:
If people visit these links they'll be at the start of the page, instead of very near the start of the page. In both cases they're still looking at the page intro.

Reply:
Just grepped the whole site, we don't have existing reference to this section. So this is fine.